ccsdspy/decode.py

```diff
@@ -1,12 +1,92 @@
 """Internal decoding routines."""
 from __future__ import division
 from collections import namedtuple
+from pickle import bytes_types
 
 import numpy as np
 
+from ccsdspy.constants import (
+    BITS_PER_BYTE,
+    PRIMARY_HEADER_NUM_BYTES,
+)
+
 __author__ = "Daniel da Silva <mail@danieldasilva.org>"
 
 
+def _get_packet_total_bytes(primary_header_bytes):
+    """Parse the  number of bytes in a packet from the bytes associated
```

Comment:
```suggestion
    """Parse the number of bytes in a packet from the bytes associated
```
There are a few places throughout this code where extra spaces show up between words in text.